### ISSUE TITLE
Devise a way to avoid is_travis() call in test_login.py

### ISSUE THREAD

**Issue by aberdean** [ole-vi/BeLL-Apps#23]
Needs some code refactoring.

**Comment by aberdean** [comment]
Worked on it repeatedly, but it still seems a tricky issue.

**Comment by dogi** [comment]
in https://github.com/ole-vi/BeLL-Apps/blob/dev/tests/test_login.py

```
    def test_login_logout(self):
        """ Test the functionality of logging in, logging out, and 
        adding the configuration to a new nation
        """
        input = ("admin", "password")
        # if remote, test first login, otherwise test second login
        if is_travis():
            self.login_test(*input, True)
            self.configuration_test()
            self.logout_test()
            self.login_test(*input)
        else:
            self.login_test(*input)
            self.logout_test()
```

as you can see in the code sample we need right now the if then call with `is_travis()` make 2 different ways for the same test

this means we have to write 2 test, which if possible in the future we can aviod ...

**Comment by aberdean** [comment]
addressed with PR #31, substituted by issue #29
